### PR TITLE
Add plain ruby docs

### PR DESCRIPTION
**Pull request by Crisfole**
Document how to use Rollbar without Rack or Rails

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,34 @@ class MyApp < Sinatra::Base
...
+  # Other Configuration Settings
+end
+
+begin
+  Rollbar.debug "Running Script"
+
+  run_script ARGV
+
+  Rollbar.info "Script ran successfully"
+rescue Exception => e
```
Comment (jondeandres) [inline README.md]: Since rescuing `Exception` is a bad practice, let's remove it.

You can do just `rescue => e` here.

Comment (Crisfole) [inline README.md]: http://stackoverflow.com/a/10048406/456188

Better yet, let's rethrow afterward

Comment (jondeandres) [inline README.md]: :+1:

**Comment by Crisfole** [comment]
@jondeandres Fixed

File: README.md [hunk 2]
```diff
@@ -97,6 +97,35 @@ class MyApp < Sinatra::Base
...
+```ruby
+require 'rollbar'
+
+Rollbar.configure do |config|
+  config.access_token = "SERVER_POST_ACCESS_TOKEN"
+  # Other Configuration Settings
+end
+
+begin
+  Rollbar.debug "Running Script"
```
Comment (jondeandres) [inline README.md]: @Crisfole I think that conceptually we should left the the `.debug()` and `.info()` calls outside the `begin/rescue` block.

File: README.md [hunk 3]
```diff
@@ -97,6 +97,35 @@ class MyApp < Sinatra::Base
...
+
+  run_script ARGV
+
+  Rollbar.info "Script ran successfully"
+rescue Exception => e # Never rescue Exception *unless* you re-raise in rescue body
+  Rollbar.log e
```
Comment (jondeandres) [inline README.md]: I think it'd be desirable having here `Rollbar.error()` insead of `Rollbar.log`.

File: README.md (hunk 2 above)
Comment (Crisfole) [inline README.md]: done

File: README.md (hunk 3 above)
Comment (Crisfole) [inline README.md]: done

**Comment by Crisfole** [comment]
@jondeandres Should be good to go.

File: README.md [hunk 4]
```diff
@@ -97,6 +97,35 @@ class MyApp < Sinatra::Base
...
+
+ 1. Require rollbar
+ 2. Configure rollbar
+ 3. Send Rollbar data
+
+```ruby
+require 'rollbar'
+
+Rollbar.configure do |config|
+  config.access_token = "SERVER_POST_ACCESS_TOKEN"
```
Comment (brianr) [inline README.md]: This should be "POST_SERVER_ITEM_ACCESS_TOKEN" to be consistent with the rest of this file (and work with the replacements that happen in our logged-in site)

**Pull request merged**